noxfile.py

```diff
@@ -40,7 +85,11 @@ def set_environment_variables(env_dict, session):
 
 @nox.session(name="pybamm-requires")
 def run_pybamm_requires(session):
-    """Download, compile, and install the build-time requirements for Linux and macOS. Supports --install-dir for custom installation paths and --force to force installation."""
+    """Download, compile, and install the build-time requirements for Linux and macOS.
+
+    Supports --install-dir for custom installation paths and --force to force
+    installation.
+    """
```

Comment:
`nox` has an annoyance (https://github.com/wntrblm/nox/issues/694) where `nox --list` prints out just the first line of the docstring when trying to list the details of all sessions. Could we please keep this in a single line?